src/components/templates/PartyMap/PartyMapPage.tsx

```diff
@@ -34,8 +32,6 @@ const PartyMap = () => {
     partygoers: state.firestore.ordered.partygoers,
   }));
 
-  useUpdateLocationEffect(user, "Map");
```

Comment:
Looks like this will break the location on party map template venues? (which aren't in use for the burn of course)

Just wondering- did this go away because we now have a more sophisticated location hook?

Reply:
Yes. Not enough time to track rooms in party maps. Just tracking venues (which party map is an instance of)